src/components/Editor/Editor.css

```diff
@@ -1,6 +1,9 @@
 .editor-wrapper {
   --debug-line-background: rgba(226, 236, 247, 0.5);
   --debug-line-border: rgb(145, 188, 219);
+  --editor-searchbar-height: 27px;
+  --editor-second-searchbar-height: 27px;
```

Comment:
i know this is copy pasted, but do we need two separate declarations of the same value? 🤔